src/sentry/integrations/slack/message_builder/issues.py

```diff
@@ -195,6 +221,26 @@ def get_tags(
     return fields
 
 
+def get_context(group: Group) -> str:
+    context_text = ""
+    context = {
+        "Events": get_group_global_count(group),
+        "Users Affected": group.count_users_seen(),
+        "State": SUBSTATUS_TO_STR.get(group.substatus, "").replace("_", " ").title(),
+        "First Seen": time_since(group.first_seen),
+    }
+    if group.issue_type in REGRESSION_PERFORMANCE_ISSUE_TYPES:
+        # another short term solution for non-error issues notification content
+        del context["Events"]
+        del context["Users Affected"]
```

Comment:
@ceorourke For regression issues we should not have any context at all. The State and First Seen date are not very useful here- we'll have to change that in the future

Reply:
👍 ok I'll remove it entirely for those